manifest: openthread: regular openthread upmerge `37fb770`

maciejbaczmanski opened the pull request:
Regular OpenThread update to `37fb770`

Signed-off-by: Maciej Baczmanski <maciej.baczmanski@nordicsemi.no>

NordicBuilder commented:
The following west manifest projects have been modified in this Pull Request:

| Name | Old Revision | New Revision | Diff |
| ---- | ------------ | ------------ |------|
| openthread | https://github.com/nrfconnect/sdk-openthread/commit/11a38a63deb6828345097036b68775ce77506886 | https://github.com/nrfconnect/sdk-openthread/commit/37fb77098982d17555dd7a3f58832714bb9df56e (main) | [nrfconnect/sdk-openthread@11a38a63..37fb7709](https://github.com/nrfconnect/sdk-openthread/compare/11a38a63deb6828345097036b68775ce77506886..37fb77098982d17555dd7a3f58832714bb9df56e) |
| zephyr | https://github.com/nrfconnect/sdk-zephyr/commit/01e2e4c9ce8294d74032a2d33e0dcb1b74802568 | https://github.com/nrfconnect/sdk-zephyr/pull/1233 | [nrfconnect/sdk-zephyr#1233/files](https://github.com/nrfconnect/sdk-zephyr/pull/1233/files) |

*Note: This message is automatically posted and updated by the Manifest GitHub Action.*

NordicBuilder commented:
## Test specification

### [CI/Jenkins/NRF](https://jenkins-ncs.nordicsemi.no/job/latest/job/sdk-nrf/job/PR-11735/1/)
- Integration Platforms

### [CI/Jenkins/integration](https://jenkins-ncs.nordicsemi.no/job/latest/job/sdk-nrf/job/PR-11735/1/)
| Test Module | File based changes | Manually selected | West overwrite |
| ----------- | :----------------: | :---------------: | :------------: |
| desktop52_verification | X | | |
| test-fw-nrfconnect-ble | X | | |
| test-fw-nrfconnect-ble_samples | X | | |
| test-fw-nrfconnect-chip | X | | |
| test-fw-nrfconnect-fem | X | | |
| test-fw-nrfconnect-nfc | X | | |
| test-fw-nrfconnect-nrf-iot_cloud | X | | |
| test-fw-nrfconnect-nrf-iot_lwm2m | X | | |
| test-fw-nrfconnect-nrf-iot_thingy91 | X | | |
| test-fw-nrfconnect-nrf_crypto | X | | |
| test-fw-nrfconnect-rpc | X | | |
| test-fw-nrfconnect-rs | X | | |
| test-fw-nrfconnect-tfm | X | | |
| test-fw-nrfconnect-thread | X | | |
| test-sdk-find-my | X | | |
| test-sdk-homekit | X | | |

**test-fw-nrfconnect-thread:** added because there was no .github/test-spec.yml in 'openthread'

[Detailed information of selected test modules](https://jenkins-ncs.nordicsemi.no/job/latest/job/sdk-nrf/job/PR-11735/1/artifact/test-spec/test-spec-summary.yml/*view*/)

*Note: This message is automatically posted and updated by the CI*

rlubos commented:
@maciejbaczmanski Can you rebase? Downstream jobs will fail due to conflict.

NordicBuilder commented:
You can find the documentation preview for this PR at this <a href='http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-11735
'>link</a>. It will be updated about 10 minutes after the documentation build succeeds.

*Note: This comment is automatically posted by the Documentation Publishing GitHub Action.*

rlubos commented:
@maciejbaczmanski The CI failures are due to yet another PR being merged in the morning (https://github.com/nrfconnect/sdk-zephyr/pull/1218), I wasn't aware it was merged. So in order to sort this out, we need yet another rebase (I've updated the manifest just now).

maciejbaczmanski commented:
> @maciejbaczmanski The CI failures are due to yet another PR being merged in the morning ([nrfconnect/sdk-zephyr#1218](https://github.com/nrfconnect/sdk-zephyr/pull/1218)), I wasn't aware it was merged. So in order to sort this out, we need yet another rebase (I've updated the manifest just now).

Rebased. Thanks for keeping an eye on that

rlubos commented:
@maciejbaczmanski It seems that something might be wrong with this Thread update? Thread tests failed three times in a row in the same way, so I start to suspect it's not just test instability.

maciejbaczmanski commented:
> @maciejbaczmanski It seems that something might be wrong with this Thread update? Thread tests failed three times in a row in the same way, so I start to suspect it's not just test instability.

Yes, I've just found the problem.  -DOT_OPERATIONAL_DATASET_AUTO_INIT=ON while building posix application is missing. I'm working on a fix for that

maciejbaczmanski commented:
Rebased. Issue has been fixed in tests implementation

rlubos commented on west.yml:
```diff
@@ -59,7 +59,7 @@ manifest:
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 35510c3d8e95826803ffaee1bb678163e1846320
+      revision: pull/1233/head
```
`eb28632680089351c7bd4365cf3742bf7eff70d3`
